crates/gen-host-js/src/lib.rs

```diff
@@ -313,10 +333,20 @@ impl Js {
             files.push(&format!("{dir}/{name}.d.ts"), gen.src.ts.as_bytes());
         }
 
-        uwriteln!(
-            self.src.ts,
-            "import {{ {camel} as {camel}{extra} }} from \"./{dir}/{name}\";"
-        );
+        // TODO: instance mode imports typings
```

Comment:
Note to handle this before merging

Reply:
For now I've just exported the imports typings so they are at least available.